src/Tribe/Field.php

```diff
@@ -68,6 +68,95 @@ class Tribe__Field {
 		 */
 		public $settings;
 
+		/**
+		 * @var string
+		 */
+		public $type;
```

Comment:
These should really all be:
```suggestion
		public string $type;
```

Reply:
I don't think it's super safe - I intentionally avoided type casting the props just to not run into edge case fatals I may miss in this fast pass. I saw some that were null/string and others I wasn't sure about. I think we want safe first at this point.

Reply:
Fair, I'll grudgingly allow it :wink: